psycopg/pqpath.c

```diff
@@ -40,7 +40,14 @@
 #include "psycopg/pgtypes.h"
 #include "psycopg/error.h"
 
+#include "postgres_fe.h"
+#include "access/xlog_internal.h"
+#include "common/fe_memutils.h"
```

Comment:
Oh, apparently I wanted to push this a bit too much :-p  I've committed a fix, it requires copying a number of postgres-internal typedefs and const, though.